### ISSUE TITLE
Still maintained?

### ISSUE THREAD

**Issue by alexpchin** [naoufal/react-native-payments#182]
Hi all, is this package still maintained?

**Comment by ArminArdekani** [comment]
Yeah, I tried setting it up but it seems it's not working. Getting unhandled promise rejects and nullpointers as described in the open issues. Gonna go with Expo SDK's payments component.

**Comment by ChrisFromCanada** [comment]
Also, the GooglePay implementation is deprecated and will be unusable as of end of September 2019.

smarkets has a fix but the PR hasn't been merged.

**Comment by geekazoid-at** [comment]
Do you guys have any suggestions for accepting apple and Google payments from within a RN app. Thank you! 👋
